### PR TITLE
Address page

### PR DESCRIPTION
**Pull request by Yamaguchi**
- Show transactions associated with the address
- Connect the esplora API
- UI improvement in the address page.

### CODE REVIEW

**Review by azuchi — COMMENTED** [review]
docker-compose.yml still use electrs-tapyrus.

**Comment by azuchi** [comment]
When you hover the mouse cursor over the copy address icon, it should be the same as the link.
Screenshot from 2021-03-12 14-41-16

File: backend/actions/address_detail.js [hunk 1]
```diff
@@ -12,47 +12,46 @@ app.use((req, res, next) => {
...
+  // TODO: This API only supports returning 25 records for one request.
+  const lastSeenTxid = req.query.lastSeenTxid;
+  const address = req.params.address;
 
-  const regex = new RegExp(/^[0-9a-zA-Z]{26,35}$/);
-  const urlAddress = req.params.address;
-
-  if (!regex.test(urlAddress)) {
-    logger.error(`Regex Test didn't pass for URL - /address/${urlAddress}`);
+  if (!/^[0-9a-zA-Z]{26,35}$/.test(address)) {
```
Comment (azuchi) [inline backend/actions/address_detail.js:19]: The address should be verified including checksum of Base58check encoding and version byte.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: backend/actions/address_detail.js (hunk 1 above)
Comment (Yamaguchi) [reply backend/actions/address_detail.js]: Use tapyrus.address.fromBase58Check function (9f7301fa93b854bbf1966a272ced505382666df8)

**Pull request merged**